wrappers/mity/report/wrapper.py

```diff
@@ -0,0 +1,23 @@
+from snakemake.shell import shell
+import os
+import process_report
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+family = snakemake.wildcards.family
+outdir = snakemake.params.outdir
+prefix = snakemake.params.prefix
+tool = snakemake.params.tool
+
+mod1 = " module load python; "
```

Comment:
Same comment as above- we shouldn't be loading modules here as they are system specific. All tools we use in a rule should be loaded into the conda environment, or specified via a path in the config.yaml.